File: TransCoder_tokenized_test_set_functions/cpp/SORTING_USING_TRIVIAL_HASH_FUNCTION.cpp
void sortUsingHash ( int a [ ], int n ) {
  int max = * std :: max_element ( a, a + n );
  int hash [ max + 1 ] = {
    0 };
    for ( int i = 0;
    i < n;
    i ++ ) hash [ a [ i ] ] += 1;
    for ( int i = 0;
    i <= max;
    i ++ ) {
      if ( hash [ i ] ) {
        for ( int j = 0;
        j < hash [ i ];
        j ++ ) {
          cout << i << " ";
        }
      }
    }
  }

File: results/java_codex/SORTING_USING_TRIVIAL_HASH_FUNCTION.java
void sortUsingHash(int a[], int n) {
  int max = *max_element(a, a + n);
  int hash[max + 1] = {0};
  for (int i = 0; i < n; i++) hash[a[i]] += 1;
  for (int i = 0; i <= max; i++) {
    if (hash[i] != 0) {
      for (int j = 0; j < hash[i]; j++) {
        cout << i << " ";
      }
    }
  }
}